### PR TITLE
Console reporter: if statistic produces percents, format it as such

### PR DESCRIPTION
**Pull request by LebedevRI**

### CODE REVIEW

**Comment by LebedevRI** [comment]
Now with tests!

File: src/console_reporter.cc [hunk 1]
```diff
@@ -161,10 +161,17 @@ void ConsoleReporter::PrintRunData(const Run& result) {
   for (auto& c : result.counters) {
     const std::size_t cNameLen = std::max(std::string::size_type(10),
                                           c.first.length());
-    auto const& s = HumanReadableNumber(c.second.value, c.second.oneK);
+    std::string s;
     const char* unit = "";
-    if (c.second.flags & Counter::kIsRate)
-      unit = (c.second.flags & Counter::kInvert) ? "s" : "/s";
+    if (result.run_type == Run::RT_Aggregate &&
```
Comment (dominichamon) [inline src/console_reporter.cc:166]: i think we repeat these two lines in a few places now... worth having a utility method on `result`?

**Review by dominichamon — APPROVED** [review]

File: src/console_reporter.cc (hunk 1 above)
Comment (LebedevRI) [reply src/console_reporter.cc:166]: The formatting here is different, there is no space separator before the unit,
and the time unit (seconds) isn't printed in the unit specified for the benchmark,
but in an automatically picked one. So i'm not sure yet how to refactor this.

**Comment by LebedevRI** [comment]
@dominichamon thank you for the review!

File: src/console_reporter.cc (hunk 1 above)
Comment (dominichamon) [reply src/console_reporter.cc:166]: i meant the conditional (`RT_Aggregate` && `kPercentage`).

**Pull request merged**